### ISSUE TITLE
Occlusion Array?

### ISSUE THREAD

**Issue by ysharma1126** [google-research/deeplab2#77]
In reference to the following [comment](https://github.com/google-research/deeplab2/issues/39#issuecomment-891356647) in a previously closed issue. 

For the occlusion array, how was the forward-backward consistency check implemented? 
Was eq. (5) in [Sundaram et al., 2010](https://lmb.informatik.uni-freiburg.de/Publications/2010/Bro10e/sundaram_eccv10.pdf) used? If so, were the same hyperparameters used, or were they tuned, and if so, how?

**Comment by markweberdev** [comment]
Hi @ysharma1126,

The cyclic consistency that we used is somewhat similar to the one we mentioned. Unfortunately, I cannot provide you the exact details/hyper-parameters used, as we did not spend time on tuning this. The idea is to find pixels that are likely occluded. So, a simple forward-backward consistency check to compute pixels that likely are occluded is used for our tracker. I hope that helps.

Best,
Mark
